validphys2/examples/data_theory_comparison_w_sv.yaml

```diff
@@ -7,8 +7,6 @@ use_cuts: "internal"
 
 theoryid: 717 # define the central theory
 point_prescription: "3 point"
-theoryids:
```

Comment:
Does this runcard work even if you don't have a list?

Reply:
Yes, all that changed is that now `theory_covmat_custom` (the function that creates the covmat for `vp-setupfit`) is collected over point_prescriptions, but most other actions still take a regular point_prescription as input so their functionality hasn't changed.

Reply:
So the reason the old runcards cannot run is just because the `from_` stuff is gone?

Reply:
Yes it would clash with the produce_theoryids that this PR introduces